tests/test_pipelines.py

```diff
@@ -432,3 +436,52 @@ def test_lms_stable_diffusion_pipeline(self):
         assert image.shape == (1, 512, 512, 3)
         expected_slice = np.array([0.9077, 0.9254, 0.9181, 0.9227, 0.9213, 0.9367, 0.9399, 0.9406, 0.9024])
         assert np.abs(image_slice.flatten() - expected_slice).max() < 1e-2
+
+    @slow
+    @unittest.skipIf(torch_device == "cpu", "Stable diffusion is supposed to run on GPU")
+    def test_stable_diffusion_img2img_pipeline(self):
+        ds = load_dataset("hf-internal-testing/diffusers-images", split="train")
+
+        init_image = ds[1]["image"].resize((768, 512))
```

Comment:
@pcuenca @anton-l @patil-suraj I think a nice way of testing a model is to just verify that the image stays the same (as this is what the user expects to see). @pcuenca added you to the org as it might be useful for you when testing Flax models :-)

So in this folder I've uploaded the input and expected output images for the two pipelines respectively :-)

Reply:
https://huggingface.co/datasets/hf-internal-testing/diffusers-images